haskala/modules/custom/haskala_general/haskala_general.features.user_permission.inc

```diff
@@ -10,24 +10,24 @@
 function haskala_general_user_default_permissions() {
   $permissions = array();
 
-  // Exported permission: 'search content'.
-  $permissions['search content'] = array(
-    'name' => 'search content',
+  // Exported permission: 'access haskala search_api_page'.
```

Comment:
permissions shouldn't be exported by features, it should be done via the ``.profile``, so an admin may change them without "overriden features" appearing. Example:

https://github.com/Gizra/negawatt-server/blob/master/negawatt/negawatt.profile#L28-L31
https://github.com/Gizra/negawatt-server/blob/master/negawatt/negawatt.profile#L72-L91